src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -117,7 +122,7 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
     const dayStart = addDays(startOfToday(), selectedDayIndex);
     const daysEvents = relatedVenueEvents
       .filter(isEventWithinDate(selectedDayIndex === 0 ? Date.now() : dayStart))
-      .map(prepareForSchedule(dayStart, userEventIds));
+      .map(prepareForSchedule(dayStart, userEventIds, "web"));
```

Comment:
There will be some minimal changes required here based on other suggestions in this review (mostly around the `prepareForSchedule` props/args)

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1423/commits/ec68e834fab42e9549a285d7a454e4c29e1f24e4